docs/source/onboarding_resources/tutorial/index.ipynb

```diff
@@ -0,0 +1,1653 @@
+{
```

Comment:
<p>In this graph, only underweight and stunting go to malaria, but in the code above, all 3 CGF seem to impact malaria?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1579/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
As we found out on Slack yesterday, this is apparently a result of `gbd_mapping` reporting which risk-cause pairs were modeled, not which pairs had _at least a 1-star association_. We'll investigate this more in the future but for the purposes of this tutorial, I'll just update the diagram to agree with `gbd_mapping` and ignore the issue.